### PR TITLE
function clear() -- Set populated arrays to empty arrays

### PR DESCRIPTION
**Pull request by craiglondon**
Fix for issues found by Scrutinizer CI
https://scrutinizer-ci.com/g/yiisoft/yii2/issues/master/files/framework/web/View.php?selectedLabels%5B0%5D=9&orderField=issueCount&order=desc&honorSelectedPaths=0

`It seems like null of type null is incompatible with the declared type array of property $metaTags.`
`It seems like null of type null is incompatible with the declared type array of property $linkTags.`
`It seems like null of type null is incompatible with the declared type array of property $css.`
`It seems like null of type null is incompatible with the declared type array of property $cssFiles.`
`It seems like null of type null is incompatible with the declared type array of property $js.`
`It seems like null of type null is incompatible with the declared type array of property $jsFiles.`

### CODE REVIEW

**Comment by SilverFire** [comment]
https://github.com/yiisoft/yii2/blob/master/framework/web/View.php#L106-L131

These default should be changed to array as well

**Comment by craiglondon** [comment]
@SilverFire I added another commit with your suggestion.

**Review by SilverFire — APPROVED** [review]

File: framework/web/View.php [hunk 1]
```diff
@@ -236,15 +236,16 @@ public function setAssetManager($value)
 
     /**
      * Clears up the registered meta tags, link tags, css/js scripts and files.
+     * @return void
```
Comment (cebe) [inline framework/web/View.php]: this is not needed.

**Review by cebe — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Review by klimov-paul — APPROVED** [review]

**Comment by klimov-paul** [comment]
Thank you.

**Pull request merged**